staticfiles/js/worker/services/worker.service.js

```diff
@@ -29,16 +29,16 @@
         url: 'http://share-quick.com/cr/getWorkerProfile.php',
         method: 'POST'
       });
-      
-    }   
+
+    }
 
     function getWorkerTaskPortfolio() {
       return $http({
-        url: 'https://api.myjson.com/bins/q7jc',
+        url: '/api/worker/1/portfolio/',
         method: 'GET'
       });
```

Comment:
The problem is solved , if I put 'return' before $http(settings).success(function (data,config) {
        deferred.resolve(arguments); in http.service.js